src/styles.scss

```diff
@@ -175,3 +175,5 @@ $deprecations--message: 'Deprecated code was found, this code will be removed be
 // carbon-addons-iot-react components
 //-------------------------------------
 @import './components/AddCard/add-card';
+@import './components/ProgressIndicator/progress-indicator';
+@import './components/SideNav/side-nav';
```

Comment:
We should add these to the rollup config to be exported/available individually

like 
https://github.com/IBM/carbon-addons-iot-react/pull/432/files#diff-ff6e5f22a9c7e66987b19c0199636480R134